plugin/pkg/scheduler/factory/factory_test.go

```diff
@@ -302,3 +307,49 @@ func TestBackoff(t *testing.T) {
 		t.Errorf("expected: 1, got %s", duration.String())
 	}
 }
+
+// TestPodFitScheduler tests if a pod with certain annotation should be picked
```

Comment:
I would write this as "tests if a pod with an annotation that should cause it to be picked by the default scheduler, is in fact picked by the default scheduler"